charts/spire/ci/external-mysql-values.yaml

```diff
@@ -0,0 +1,11 @@
+spire-server:
+  dataStore:
+    sql:
+      databaseType: mysql
+      databaseName: spire-server
+      username: spire
+      password: sp1ff3Test
+      host: mysql.mysql
+      port: 3306
+      options:
+      - parseTime: true
```

Comment:
We need to be careful including stuff in the chart. it gets loaded all the way into the users cluster. Maybe the .helmignore file is enough to exclude this file though.